### DISCUSSION TITLE
Is there a showcases page?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#5262] [Archive]
Is there a showcases page?

**Comment by benhaynes** [comment]
No, not as of now.

**Reply by biapar** [reply]
Which are the Directus point of force respect to Strapi?

**Reply by benhaynes** [reply]
Both are open-source headless CMS (though Strapi has paywalls, and Directus does not), but the main difference is that Directus can be installed on top of any existing SQL database to manage it. Strapi by comparison can save to SQL but you manually have to define the schema.

There are many _many_ feature differences, far too many to get into here. You're best to try both and compare for your use case.

**Reply by paescuj** [reply]
This could be something for the new [awesome-directus](https://github.com/directus-community/awesome-directus) list.

**Reply by biapar** [reply]
Thanks for your reply. I think that the content on website must be better organized. In the application cases must be putted the real cases, create some pages about on how to use with Nuxt/Next and so, mobile app, how to use Directus in the common tasks like use API with JWT or with SSO, extend APIs,  etc..
I use Strapi too.
How Can I help or collaborate with you?

**Reply by benhaynes** [reply]
Hey @biapar — we _just_ hired two technical content writers who will help us get some of these integrations (eg: Gatsby, Nuxt, etc) written. However, we'd love all the help we can get! If you have ideas for pages, tutorials, articles, etc... I'd love to chat further! You can DM me on our community Discord: https://directus.chat

**Reply by biapar** [reply]
> Hey @biapar — we _just_ hired two technical content writers who will help us get some of these integrations (eg: Gatsby, Nuxt, etc) written. However, we'd love all the help we can get! If you have ideas for pages, tutorials, articles, etc... I'd love to chat further! You can DM me on our community Discord: https://directus.chat

Hi, Is the technical content writer job full or contract type? I'll contact you on Discord.
